### PR TITLE
Fix: Over-restricting of the `/admin` endpoint

### PR DESCRIPTION
**Pull request by nss10**
Link to Jira ticket : [BRH-440](https://ctds-planx.atlassian.net/browse/BRH-440)

### Bug Fixes
* `/admin` endpoint has been over-restricted in the previous merge. Even Admin's were being forcefully redirected. 

[BRH-440]: https://ctds-planx.atlassian.net/browse/BRH-440?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mfshao** [comment]
logically it is fine, but there are merge confilcts that needs to be addressed

**Review by aartivnkt — APPROVED** [review]

File: bmh_admin_portal_ui/src/components/admin-route.jsx [hunk 1]
```diff
@@ -12,20 +12,39 @@ import {
...
+      <Route
+        {...rest}
+        render={props =>
+          <Redirect to={{ pathname: '/', state: { from: props.location } }} />
+        }
+      />
+    )
+  }
+
+  if(adminAuthorized===null){ //Only return <div/> if adminAuthorized equal to null, not when equal to false.
```
Comment (mfshao) [inline bmh_admin_portal_ui/src/components/admin-route.jsx:38]:
```suggestion
  if(isLoggedIn && adminAuthorized===null){ //Only return <div/> if adminAuthorized equal to null, not when equal to false.
```
if you do this then you can merge the logic from L27-L36 into the block below?

Comment (nss10) [reply bmh_admin_portal_ui/src/components/admin-route.jsx]: Yeah! That seemed like a nice idea. I just implemented and tested this change.

**Review by mfshao — APPROVED** [review]
👍

**Pull request merged**